content/02.main-text.md

```diff
@@ -211,6 +211,19 @@ manubot cite --render --format=markdown \
   pmid:29618526 doi:10.1038/550143a
 ```
 
+### Interactive features and appearance
+
+Manubot comes with several "plugins" that can be included in manuscripts exported as HTML.
+These plugins add special interactive features that enhance the user experience of viewing and reading manuscripts (@fig:plugins).
+The build process can also accommodate different "themes", which change the general aesthetics and appearance of the exported document (for example, from a contemporary sans-serif style to a more traditional serif style).
+
+![
+Screenshots of the various Manubot plugins, illustrating their functionality and usefulness.
+](images/plugins.png "Manubot Plugins"){#fig:plugins}
+
+The Manubot "front-end" (layout, look, controls, behavior, etc.) is designed to be in line with current best practices and user expectations of the modern web.
+In addition, the plugins utilize standard technology built in to (almost) all major web browsers, allowing them to be relatively lightweight, modular, and easy to configure.
```

Comment:
Perhaps there is some way we can note that these plugins/themes assumes pandoc-generated HTML and thus can be used by the large body of projects that use pandoc.

Reply:
We could say that. I'd want to test it on some other just-pandoc manuscripts before claiming that though. There are certain assumptions it makes, like the fignos filters, and the --section parameter, and things like that.

Reply:
Makes sense. So let's hold off on this assertion until we have time to evaluate it. (which may not be until after submitting the meta review).